projects/1.20/assets/food-expansion-reimagined/foodexpansion/lang/zh_cn.json

```
@@ -0,0 +1,44 @@
+{
+    "item.foodexpansion.jelly": "翡翠果冻",
+    "item.foodexpansion.bacon": "培根",
+    "item.foodexpansion.cooked_bacon": "香脆培根",
+    "item.foodexpansion.fried_egg": "煎蛋",
+    "item.foodexpansion.bacon_and_egg": "培根煎蛋",
+    "item.foodexpansion.carrot_seed_soup": "胡萝卜汤",
+    "item.foodexpansion.squid": "生鱿鱼",
+    "item.foodexpansion.cooked_squid": "烤鱿鱼",
+    "item.foodexpansion.compressed_flesh": "腊肉",
+    "item.foodexpansion.chocolate_bar": "巧克力棒",
+    "item.foodexpansion.spider_soup": "蛛眼炖汤",
+    "item.foodexpansion.nether_wart_soup": "下界疣炖菜",
+    "item.foodexpansion.cactus_fruit": "仙人掌果",
+    "item.foodexpansion.horse_meat": "生马肉",
+    "item.foodexpansion.cooked_horse_meat": "熟马肉",
+    "item.foodexpansion.forbidden_fruit": "§e禁果",
+    "item.foodexpansion.cooked_mushroom": "烤蘑菇",
+    "item.foodexpansion.carrot_pie": "胡萝卜蛋糕",
+    "item.foodexpansion.bat_wing": "生蝙蝠翅膀",
+    "item.foodexpansion.cooked_bat_wing": "烤蝙蝠翅膀",
+    "item.foodexpansion.blaze_cream": "烈焰香奶汤",
```

```suggestion
    "item.foodexpansion.blaze_cream": "烈焰奶油汤",
```
没必要多此一举，[奶油汤](https://zh.wikipedia.org/zh-cn/%E5%BF%8C%E5%BB%89%E6%B9%AF)就是奶油汤
同样的，如果`Soup`有其他对应，请包括这里进行全文通查